webapp/channels/src/components/signup/signup.test.tsx

```diff
@@ -295,4 +297,53 @@ describe('components/signup/Signup', () => {
             expect(wrapper.find('.content-layout-column-title').text()).toEqual('This invite link is invalid');
         });
     });
+
+    it('should show newsletter check box opt-in for self-hosted non airgapped workspaces', async () => {
```

Comment:
Same as on another PR with a frontend piece, even though this is an existing test file, for the new tests could we use `@testing-library/react` since `jest` is deprecated?